llvm/docs/MLGO.rst

```diff
@@ -490,24 +490,14 @@ embeddings can be computed and accessed via an ``ir2vec::Embedder`` instance.
       std::unique_ptr<ir2vec::Embedder> Emb = std::move(*EmbOrErr);
 
 3. **Compute and Access Embeddings**:
```

Comment:
Prompted by this change - you could clarify that the validity of the Embedder is tied to the function not being changed, i.e. it's meant to be used (and invalidated) as part of a "hosting" function analysis.